### PR TITLE
Added CockroachDB to list of third-party DB backends.

### PR DESCRIPTION
**Pull request by carltongibson**
I expected to find this here when looking for it today, so...

### CODE REVIEW

File: docs/ref/databases.txt [hunk 1]
```diff
@@ -1056,17 +1056,17 @@ Using a 3rd-party database backend
...
-* Firebird_
-* ODBC_
+* `ODBC`_
 
 The Django versions and ORM features supported by these unofficial backends
 vary considerably. Queries regarding the specific capabilities of these
 unofficial backends, along with any support queries, should be directed to
 the support channels provided by each 3rd party project.
 
+.. _Firebird: https://github.com/maxirobaina/django-firebird
```
Comment (pope1ni) [inline docs/ref/databases.txt:1071]: Update to point to https://pypi.org/project/django-firebird/?

File: docs/ref/databases.txt [hunk 2]
```diff
@@ -1056,17 +1056,17 @@ Using a 3rd-party database backend
...
 .. _IBM DB2: https://pypi.org/project/ibm_db_django/
 .. _Microsoft SQL Server: https://pypi.org/project/django-pyodbc-azure/
-.. _Firebird: https://github.com/maxirobaina/django-firebird
 .. _ODBC: https://github.com/lionheart/django-pyodbc/
```
Comment (pope1ni) [inline docs/ref/databases.txt:1074]: Update to point to https://pypi.org/project/django-pyodbc/?

**Comment by timgraham** [comment]
Incidental to the addition of CockroachDB, generally we've required the libraries listed here to be actively maintained.
- ibm_db_django looks abandoned (no releases since 2018)
- django-pyodbc-azure has moved to https://pypi.org/project/django-mssql-backend/
- django-firebird seems to only support Django 1.11.x. 
- django-pyodbc doesn't have releases since 2017.

**Comment by carltongibson** [comment]
Thanks Tim. I updated the SQL Server URL. 

There seems to be recent activity on both [ibm_db_django](https://github.com/ibmdb/python-ibmdb-django/issues/49) and [django-firebird](https://github.com/maxirobaina/django-firebird/issues/110) from the last few days. Given that culling them wasn't on my agenda, I'll give them the benefit of the doubt for now, but I've made a note to check up on them again before 3.1 final (i.e. before we stop backporting docs fixes to stable/3.0.x).

**Pull request merged**